maths/linear_diophantine_check.py

```diff
@@ -0,0 +1,101 @@
+"""
+   Program to check for solutions
+   of linear diophantine equations
+   can be possible or not
+"""
+
+# importing inbuilt gcd function
+from math import gcd
+
+"""
+   This function checks if integral
+   solutions are possible
+   >>>a,b,c, are parameters represents
+   >>>represents the variables in equation
+   ax + by = c
+"""
+
+
+def is_possible(a, b, c) -> bool:
```

Comment:
```suggestion
def has_linear_diophantine_solutions(a: int, b: int, c: int) -> bool:
```
Make the name clearer and add function argument type hints